synapseutils/sync.py

```diff
@@ -216,19 +216,27 @@ def readManifestFile(syn, manifestFile):
             raise ValueError("Manifest must contain a column of %s" %field)
     sys.stdout.write('OK\n')
 
-    sys.stdout.write('Validating that all paths exist')
-    df.path = df.path.apply(_check_path_and_normalize)
+    if 'dataFileHandleId' in df.columns and 'path' in df.columns:
+        raise ValueError("Cannot have 'path' and 'dataFileHandleId' columns in the same manifest.")
 
-    sys.stdout.write('OK\n')
+    if 'path' in df.columns:
```

Comment:
what happens in the edge case when the 'path' header is specified in the TSV but the rest of the column is left blank?